wgsl/index.bs

```diff
@@ -2007,33 +2007,43 @@ Issue: (dneto): Bitwise-complement is under discussion.  https://github.com/gpuw
   <thead>
     <tr><td>Precondition<td>Conclusion<td>Notes
   </thead>
-  <tr><td>*e1* : *T*<br>
-          *e2* : *T*<br>
-          *T* is *Int*
-       <td class="nowrap">`e1 << e2` : *T*
-       <td>Shift *e1* left by *e2* bits (OpShiftLeftLogical)
-  <tr><td>*e1* : *T*<br>
-          *e2* : *T*<br>
-          *T* is *IntVec*
-       <td class="nowrap">`e1 << e2` : *T*
+  <tr algorithm="scalar shift left"><td>|e1| : |T|<br>
+          |e2| : u32<br>
+          |T| is *Int*
+       <td class="nowrap">|e1| `<<` |e2| : |T|
+       <td>Shift left:<br>
+           Shift |e1| left, inserting zero bits at the least significant positions,
+           and discarding the most significant bits.
+           The number of bits to shift is the value of |e2| modulo the bit width of |e1|.
+           (OpShiftLeftLogical)
+  <tr algorithm="vector shift left"><td>|e1| : vec|N|&lt;|T|&gt;<br>
+          |e2| : vec|N|&lt;u32&gt;<br>
+          |T| is *Int*
+       <td class="nowrap">|e1| `<<` |e2| : vec|N|&lt;|T|&gt;
        <td>Component-wise shift left (OpShiftLeftLogical)
```

Comment:
Should we say something like a[i] << b[i] ?

Reply:
Done. PTAL.